src/app/features/data-import/data-import/data-import.component.html

```diff
@@ -1,13 +1,49 @@
-<button
-    (click)="csvImport.click()"
-    mat-stroked-button
-    i18n="Import database button"
-  >
-    Import (.csv)
-</button>
-<input
-  #csvImport
-  type="file"
-  style="display: none"
-  (change)="importCsvFile($event.target.files[0])"
-/>
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons"rel="stylesheet">
+<mat-vertical-stepper linear #stepper>
+    <mat-step [stepControl]="firstFormGroup" [editable]="true">
+      <form [formGroup]="firstFormGroup">
+        <ng-template matStepLabel>Import Type</ng-template>
+        <mat-form-field>
+            <mat-label>Entity Type</mat-label>
+            <mat-select formControlName="firstCtrl" required (selectionChange)="entitySelectionChanged()">
+                <mat-option *ngFor="let entity of getEntitiesMap() | keyvalue" [value]="entity.key">
+                    {{entity.key}}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <div>
+          <button mat-button matStepperNext>Next</button>
+        </div>
+      </form>
+    </mat-step>
+    <mat-step [stepControl]="secondFormGroup" [editable]="true" [completed]="hasValidFile">
+      <form [formGroup]="secondFormGroup">
+        <ng-template matStepLabel>Import File</ng-template>
+        <mat-form-field>
+          <mat-label>Selected File: </mat-label>
+          <input matInput formControlName="secondCtrl" readonly placeholder="No file selected" required/>
+        </mat-form-field>
+        <div>
+          <button (click)="csvImport.click()" mat-stroked-button>
+            Select File (*.csv)
+          </button>
+          <input #csvImport type="file" style="display: none" (change)="setCsvFile($event.target.files[0])"/>
+        </div>
+        <div>
+          <button mat-button matStepperPrevious>Back</button>
+          <button mat-button matStepperNext>Next</button>
+        </div>
+      </form>
+    </mat-step>
+    <mat-step>
+      <ng-template matStepLabel>Excecute</ng-template>
```

Comment:
Agreed, got that one in my presentation that this is missing, but desinging it would be nice anyway.
- Entity Type: <Type>
- Import File: <Filename>
- Records in File: <Number>
- Transaction ID: <ID>